docs/testing/testing-with-puppeteer.md

```diff
@@ -33,19 +33,19 @@ web app, like "Type this key", "Wait until this HTML element
 appears/disappears", or "Click on this HTML element".
 
 For example, this function might test the `x` keyboard shortcut to
-open the compose box for a new private message:
+open the compose box for a new direct message:
 
 ```js
-async function test_private_message_compose_shortcut(page) {
+async function test_direct_message_compose_shortcut(page) {
```

Comment:
If we make this change (and the ones below in this file), then I would do this update in a separate commit and make the updates to the code referenced here in the codebase. Otherwise, the documentation won't be accurate.

Reply:
Oh I see, good point. I can try doing this update in a separate commit, but maybe it would make more sense to take care of this as part of the bulk conversation PR for `private_message` in the `web/` directory?

Reply:
Yeah, I can see this being part of the audit/update for `private_message` with an underscore since we're just targeting `private message` with a space in this pass. So, dropping these changes from this PR seems fine.